src/main/java/me/coley/cafedude/attribute/StackMapTableAttribute.java

```diff
@@ -0,0 +1,383 @@
+package me.coley.cafedude.attribute;
+
+/**
+ * <p>
+ * Used during the process of verification by type checking.
+ * </p>
+ * 
+ * <p>
+ * There may be at most one StackMapTable attribute in the attributes table of
+ * a Code attribute.
+ * </p>
+ * 
+ * <p>
+ * In a class file whose version number is 50.0 or above, if a method's Code
+ * attribute does not have a StackMapTable attribute, it has an implicit stack
+ * map attribute (§4.10.1). This implicit stack map attribute is equivalent to
+ * a StackMapTable attribute with number_of_entries equal to zero.
+ * </p>
+ * 
+ * @author x4e
+ */
+public class StackMapTableAttribute extends Attribute {
+	public final StackMapFrame[] frames;
+
+	/**
+	 * @param nameIndex Name index in constant pool.
+	 * @param frames An array of stack map frames.
+	 */
+	public StackMapTableAttribute(int nameIndex, StackMapFrame[] frames) {
+		super(nameIndex);
+		this.frames = frames;
+	}
+
+	@Override
+	public int computeInternalLength() {
+		// u2 number_of_entries
+		int length = 2;
+		for (StackMapFrame frame : frames) {
+			length += frame.computeCompleteLength();
+		}
+		return length;
+	}
+
+	/**
+	 * A verification type specifies the type of either one or two locations,
+	 * where a location is either a single local variable or a single operand
+	 * stack entry. A verification type consists of a one-byte tag, indicating
+	 * which type is in use, followed by zero or more bytes, giving more
+	 * information about the tag.
+	 */
+	public abstract static class TypeInfo {
+		protected int computeInternalLength() {
```

Comment:
Since this technically represents the length of the info piece and isn't the internal length of any attribute, it can be named so better match the intent. Just `computeLength()` or `getSize()` since there really isn't computation involved.